pkg/resources/testdata/TestAcc_RowAccessPolicy/basic/variables.tf

```diff
@@ -10,7 +10,7 @@ variable "schema" {
   type = string
 }
 
-variable "arguments" {
+variable "argument" {
   type = set(map(string))
```

Comment:
not that important, but if you would change MapVariable to ObjectVariable when building model this should be
```terraform
variable "argument" {
  type = set(object({
    name = string
    type = string
  }))
}
```
and test.tf's should be adjusted to use object with fields instead of map